File: Figuras/ejemplo_figuras.py
import circulo
import rectangulo
import cuadrado

def main():
    circ = circulo.Circulo(5)
    rect = rectangulo.Rectangulo(5, 7)
    cuad = cuadrado.Cuadrado(6)

    print("El círculo tiene un radio de", circ.radio)
    print("Su área es de", circ.area())
    print("Su perímetro es de", circ.perimetro(), "\n")
    print("El rectangulo tiene base", rect.base, "y altura", rect.altura)
    print("Su área es de", rect.area())
    print("Su perímetro es de", rect.perimetro(), "\n")
    print("El cuadrado tiene lado", cuad.lado)
    print("Su área es de", cuad.area())
    print("Su perímetro es de", cuad.perimetro(), "\n")

main()

File: Figuras/EjemploFiguras.java
public class EjemploFiguras {
    
    public void ejecutar() {
        Circulo circulo = new Circulo(3);
        Rectangulo rectangulo = new Rectangulo(5, 7);
        Cuadrado cuadrado = new Cuadrado(6);

        System.out.println("El círculo tiene un radio de " + circulo.radio);
        System.out.println("Su área es de " + circulo.area());
        System.out.println("Su perímetro es de " + circulo.perimetro() + "\n");
        System.out.println("El rectángulo tiene base " + rectangulo.base + " y altura " + rectangulo.altura);
        System.out.println("Su área es de " + rectangulo.area());
        System.out.println("Su perímetro es de " + rectangulo.perimetro() + "\n");
        System.out.println("El cuadrado tiene lado " + cuadrado.lado);
        System.out.println("Su área es de " + cuadrado.area());
        System.out.println("Su perímetro es de " + cuadrado.perimetro() + "\n");
    }
        
    public static void main(String[] args) {
        EjemploFiguras miEjemplo = new EjemploFiguras();
        miEjemplo.ejecutar();
    }
}
